src/javascript/core/transports/image.js

```diff
@@ -6,6 +6,7 @@ module.exports = function () {
 	return {
 		name: 'image',
 		send: function (url, data) {
+			url = url.replace('https://spoor-api.ft.com/ingest', 'https://spoor-api.ft.com/px.gif');
```

Comment:
I limited it to rewriting the ft.com ingest point only (https://spoor-api.ft.com/ingest) as I didn't want to disrupt NAR in any way (https://github.com/Financial-Times/o-tracking-builder-service/blob/master/views/products/nar.pug#L4)